cloud/services/domains.go

```diff
@@ -0,0 +1,171 @@
+package services
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/linode/linodego"
+	kutil "sigs.k8s.io/cluster-api/util"
+
+	"github.com/linode/cluster-api-provider-linode/cloud/scope"
+)
+
+var dnsTTLSec = 30
```

Comment:
if this is a default I think we should add this to `util/reconcile/defaults.go`

Reply:
done